### PR TITLE
Add ISV CloudPak guidance

### PR DESCRIPTION
**Pull request by bgower-ibm**

### CODE REVIEW

File: community/cloud-pak.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+## ibm_cloud_pak/manifest.yaml
+
+IBM Cloud Private clusters are typically in an air-gapped environment with no access to the public internet.  When installing charts that point to a public image registry like Docker Hub, the chart pods will fail to pull the images.  To help alleviate this, IBM Cloud Private provides a local image registry as well as a local chart repository as well as tooling for building and installing binary packages in an air-gapped environment.
+
+IBM Cloud Private provides tooling to [build offline binary packages](https://www.ibm.com/support/knowledgecenter/en/SSBS6K_3.1.1/manage_cluster/cli_catalog_commands.html#create-archive).  The tool reads a YAML manifest file in the helm chart and creates the binary package from the specified contents.  Cloud Paks must provide a such a manifest.yaml file for users to easily create offline binary packages in a consistent manner across products.  The manifest format can be found here: [IBM Cloud Pak manifest format](../reference/manifest-format.md)
```
Comment (cdjohnson) [inline community/cloud-pak.md]: Update the link to:  `../spec/packaging/ibm_cloud_pak/manifest-yaml.md`

File: community/cloud-pak.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+IBM Cloud Private also supports [importing binary packages that contain all the components (charts and images) required to deploy a product](https://www.ibm.com/support/knowledgecenter/en/SSBS6K_3.1.1/manage_cluster/cli_catalog_commands.html#load-archive).  The contents of the binary package are then stored in the local image registry / local chart repository and the charts can then be deployed without access to the public internet.
+
+Cloud Paks must provide a valid manifest.yaml file and test that the ICP tooling can build the binary offline package, that the package can be installed in ICP and that the product can be deployed from that installed package.
```
Comment (cdjohnson) [inline community/cloud-pak.md]: Let's add an abridged version of the qualificyation.yaml spec from our internal git:
https://github.ibm.com/IBMPrivateCloud/cloud-pak-spec/blob/master/spec/qualification-yaml.md

**Comment by bgower-ibm** [comment]
@cdjohnson:  Good points, done.

File: community/cloud-pak.md (hunk 1 above)
Comment (bgower-ibm) [reply community/cloud-pak.md]: Agreed

**Pull request merged**